man/Makefile

```diff
@@ -0,0 +1,14 @@
+Q := @
+
+clean:
+	rm -f elixir.1
+	rm -f iex.1
+
+build:
```

Comment:
Should we make build the first task in the makefile so we can just call `make`? :)